packages/aws-cdk/lib/api/hotswap/cloudwatch-logs.ts

```diff
@@ -0,0 +1,97 @@
+import * as cxapi from '@aws-cdk/cx-api';
```

Comment:
I don't think this file is in the correct directory anymore (it's not related to hotswapping).

Reply:
moved it up a level.